### ISSUE TITLE
Add 2017 SpashData List

### ISSUE THREAD

**Issue by davidclarke-au** [danielmiessler/SecLists#173]
https://13639-presscdn-0-80-pagely.netdna-ssl.com/wp-content/uploads/2017/12/Top-100-Worst-Passwords-of-2017a.pdf

**Comment by davidclarke-au** [comment]
Feel free to use this - https://gist.github.com/davidclarke-au/92dde10808ab8798558b3c918bd045a6

Just take off the other additions I made at the end...

**Comment by g0tmi1k** [comment]
I see the the extra being:

```
pageup,pageuppeople,pageuppassword,hiring,staff,dashboard,mbl,abcdefg2,abcdefg3,abcdefg1,Fairfax,jobad,graduate,holiday,12345678,23456789,34567890,01234567,pass1234,123456,12345,qwerty,123456789,1234567890,baseball,dragon,football,1234567,monkey,letmein,abc123,11111111,22222222,33333333,44444444,55555555,66666666,77777777,88888888,99999999,00000000,mustang,michael,superman,696969,123123123,trustno1,god12345,test1234,test4321,test1111,rush1234,rush2112,test2222,test3333,test4444,test5555,test6666,test7777,test8888,test9999,test0000,pass1111,pass2222,pass3333,pass4444,pass5555,pass6666,pass7777,pass,8888,pass9999,pass0000,H4!b5at+kWls-8yh4Guq
```

Whats the source of that?

**Comment by g0tmi1k** [comment]
Fixed in #180
